src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -709,14 +709,14 @@ export class CalciteSlider {
     } else if (key === "End") {
       adjustment = max;
     }
-
     if (isNaN(adjustment)) {
       return;
     }
-
     event.preventDefault();
     this[activeProp] = this.clamp(adjustment, activeProp);
-    this.emitChange();
+    if (this[activeProp] === adjustment) {
+      this.emitInput();
```

Comment:
This is not correct. Both input and change events should be emitted in this case.

Reply:
I misunderstood that we don't need to emit `change` event for keyboard interactions.